src/js/components/PodInstancesTable.js

```diff
@@ -357,9 +357,10 @@ class PodInstancesTable extends React.Component {
         data={this.getTableDataFor(instances, filterText)}
         expandAll={!!filterText}
         getColGroup={this.getColGroup}
+        itemHeight={TableUtil.getRowHeight()}
         onCheckboxChange={this.handleItemCheck}
-        sortBy={{prop: 'startedAt', order: 'desc'}}
```

Comment:
Arh! Sorry, I thought this was the Pod version of tasks. I get it now. Yes, this makes sense.